Cpp/odin-fg/native.expect.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+    Copyright 2020 Red Anchor Trading Co. Ltd.
+
+    Distributed under the Boost Software License, Version 1.0.
+    See <http://www.boost.org/LICENSE_1_0.txt>
+ */
+
+
+#include <odin/fg/native.hpp>
+
+#include <fost/insert>
+#include <fost/ua/cache>
+#include <fost/ua/cache.detail.hpp>
+
+
+const fg::frame::builtin odin::lib::expect = [](fg::frame &stack,
```

Comment:
Why do we need this here? Shouldn't this be in the fostgres repo?

Reply:
I will check